static/js/components/ProfileProgressControls.js

```diff
@@ -42,7 +45,12 @@ export default class ProfileProgressControls extends React.Component {
   };
 
   render() {
-    const { nextUrl, prevUrl, nextBtnLabel } = this.props;
+    let { nextUrl, prevUrl, nextBtnLabel, profilePatchStatus } = this.props;
+
+    let disabled = profilePatchStatus === FETCH_PROCESSING;
```

Comment:
since it's really capturing the fact that a request is currently being processed, what do you think about renaming `disabled` to be `inFlight` or similar?